io/aio-wrapper.cpp

```diff
@@ -152,8 +152,8 @@ namespace photon
     static void resume_libaio_requesters()
     {
 retry:
-        struct io_event events[IODEPTH];
-        int n = HAVE_N_TRY(my_io_getevents, (0, IODEPTH, events));
+        struct io_event events[IODEPTH_MAX];
+        int n = HAVE_N_TRY(my_io_getevents, (0, IODEPTH_MAX, events));
```

Comment:
should use iodepth here instead of IODEPTH_MAX

Reply:
using `iodepth` means here must be a dynamic sized array, might not be a good idea

Reply:
just my_io_getevents use iodepth